include/SDL3/SDL_video.h

```diff
@@ -96,6 +96,27 @@ typedef struct SDL_DisplayMode
     void *driverdata;           /**< driver-specific data, initialize to 0 */
 } SDL_DisplayMode;
 
+/**
+ * The structure containing the display/window HDR color and luminance properties.
+ *
+ * \since This struct is available since SDL 3.0.0
+ *
+ * \sa SDL_GetDisplayHDRProperties
+ * \sa SDL_GetWindowHDRProperties
+ */
+typedef struct SDL_HDRProperties
+{
+    SDL_FPoint red_primary;         /**< the red primary x/y coordinates */
+    SDL_FPoint green_primary;       /**< the green primary x/y coordinates */
+    SDL_FPoint blue_primary;        /**< the blue primary x/y coordinates */
+    SDL_FPoint white_point;         /**< the white point x/y coordinates */
+    float min_luminance;            /**< the min luminance level */
+    float max_luminance;            /**< the max luminance level */
+    float max_full_frame_luminance; /**< the max full-frame luminance */
+    float SDR_white_level;          /**< the SDR white level */
+    float HDR_headroom;             /**< the additional dynamic range that can be displayed in terms of the SDR white level (1.0 if HDR is not enabled) */
+} SDL_HDRProperties;
```

Comment:
If you already have min/max luminance and SDR white level then headroom is redundant. On the other hand, just the headroom is not sufficient to describe dynamic range in the blacks.

I suspect the primaries and min/max luminance are  describing the target color volume (https://gitlab.freedesktop.org/pq/color-and-hdr/-/blob/main/doc/mastering_display_metadata.md) and not the min/max luminances of the encoding.

The min/max luminances of the encoding must be known to make sense of the SDR white level. PQ defines this but everything else does not, which means that this needs to be communicated.

The reference white level and min/max luminance of the encoding are requirements to properly anchor content and tone map content. The target color volume is merely an optimization to reduce the mapping distance.

Really none of that is depending on "HDR"ness.

Thus, what you probably should do, is this:

```suggestion
typedef struct SDL_TargetColorVolume
{
    SDL_FPoint red_primary;         /**< the red primary x/y coordinates */
    SDL_FPoint green_primary;       /**< the green primary x/y coordinates */
    SDL_FPoint blue_primary;        /**< the blue primary x/y coordinates */
    SDL_FPoint white_point;         /**< the white point x/y coordinates */
    float min_luminance;            /**< the min luminance level */
    float max_luminance;            /**< the max luminance level */
    float max_full_frame_luminance; /**< the max full-frame luminance */
} SDL_TargetColorVolume;

typedef struct EncodingLuminances
{
    float min_luminance;            /**< the min luminance of the encoding in nits*/
    float max_luminance;            /**< the max luminance of the encoding in nits */
    float reference_white_level;          /**< the reference white level in nits */
} SDL_EncodingLuminances;
```

where SDL_EncodingLuminances is either required or implied, and SDL_TargetColorVolume is optional.